test/images/kitten/BASEIMAGE

```diff
@@ -1,8 +1,8 @@
-linux/amd64=REGISTRY/agnhost:2.14-linux-amd64
-linux/arm=REGISTRY/agnhost:2.14-linux-arm
-linux/arm64=REGISTRY/agnhost:2.14-linux-arm64
-linux/ppc64le=REGISTRY/agnhost:2.14-linux-ppc64le
-linux/s390x=REGISTRY/agnhost:2.14-linux-s390x
-windows/amd64/1809=REGISTRY/agnhost:2.14-windows-amd64-1809
-windows/amd64/1903=REGISTRY/agnhost:2.14-windows-amd64-1903
-windows/amd64/1909=REGISTRY/agnhost:2.14-windows-amd64-1909
+linux/amd64=REGISTRY/agnhost:2.16-linux-amd64
```

Comment:
@fedepaol I don't think you meant to make these changes in `test/images/kitten` and `test/images/nautilus`?

Reply:
It was intentional, but I missed updating the version to the one we are building.
Here https://github.com/kubernetes/kubernetes/issues/89782#issuecomment-608340939 I was pointed out to https://github.com/kubernetes/kubernetes/tree/master/test/images#making-changes-to-images which says
"Some test images (agnhost) are used as bases for other images (kitten, nautilus). If the parent image's VERSION has been bumped, also bump the version in the children's BASEIMAGE files in order for base image changes to be reflected in the child images as well."

Now, it looks like this is not being done so often (I rebased the agnost version but did not have to rebase these files), so either removing these or bumping up to the updated version is fine to me.

Also pinging @claudiubelu as he gave me the instruction link last time.

Reply:
Yeah, in fact, those values will have to be bumped to 2.20 (the version this PR bumps agnhost to). This is especially needed by the Image Builder / Promoter job, as it was out of order for a while, and there's no agnhost.2.16 in staging. Same about the nautilus image.